src/test/java/org/sonarsource/sonarlint/ls/mediumtests/LanguageServerWithFoldersMediumTests.java

```diff
@@ -111,6 +134,9 @@ void shouldBatchAnalysisFromTheSameFolder() {
 
     client.logs.clear();
 
+    // Deliberately slow down analysis timer to make sure that analyses triggered by change events are batched
+    AnalysisScheduler.setAnalysisTimerMs(5_000);
```

Comment:
Do we need to reset the timer at the end of this particular test?
(atm, the rest will happen @AfterAll, so other tests in this class running after this one will still use 5_000)